### PR TITLE
Rationale for spare CIDs

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]

File: draft-ietf-tls-dtls13.md [hunk 1]
```diff
@@ -1900,8 +1900,10 @@ MUST be used immediately for all future records. If it is set to
 Endpoints SHOULD use receiver-provided CIDs in the order they were provided.
 Endpoints MUST NOT have more than one NewConnectionId message outstanding.
 
-If the client and server have negotiated the "connection_id" extension,
-either side can request a new CID using the RequestConnectionId message.
+If the client and server have negotiated using the "connection_id" extension,
+either side can request a new CID using the RequestConnectionId message. In
+general, implementations SHOULD use a new CID whenever sending on a new
+path and thus attempt to have enough spare CIDs available to do so.
```
Comment (chris-wood) [inline draft-ietf-tls-dtls13.md:1906]: This confuses me. Presumably this is trying to say, "implementations should use a new CID for new paths, and should also request new CIDs if they anticipate path changes," or something. 

```suggestion
general, implementations SHOULD use a new CID whenever sending on a new path,
and SHOULD request new CIDs for this purpose if path changes are anticipated.
```

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-ietf-tls-dtls13.md [hunk 2]
```diff
@@ -1900,8 +1900,10 @@ MUST be used immediately for all future records. If it is set to
 Endpoints SHOULD use receiver-provided CIDs in the order they were provided.
 Endpoints MUST NOT have more than one NewConnectionId message outstanding.
 
-If the client and server have negotiated the "connection_id" extension,
-either side can request a new CID using the RequestConnectionId message.
+If the client and server have negotiated using the "connection_id" extension,
```
Comment (kaduk) [inline draft-ietf-tls-dtls13.md:1903]:
```suggestion
If the client and server have negotiated use of the "connection_id" extension,
```

Comment (hannestschofenig) [reply draft-ietf-tls-dtls13.md:1903]:
```suggestion
If the client and server have negotiated the use of CIDs with the "connection_id" extension,
```

**Comment by hannestschofenig** [comment]
The proposed text sounds reasonable to me.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**